### DISCUSSION TITLE
Is it possible for the browser extension to match the whole URL?

### DISCUSSION THREAD

**Discussion by nk-tedo-001** [keepassxreboot/keepassxc#11507] [Q&A] [answered]
Is it possible for the browser extension to match the whole URL?

When working with multiple projects that have the same auth mechanism, it is very inconvenient to see all logins.
We use Keycloak divided by reals.

Is it possible to customize the entry in KeePassXC or the plugin to use the full path?

For example
1. https://keycloak.xxx.com/realms/project1
2. https://keycloak.xxx.com/realms/project2

For now logins for both project1 and project2 are displayed, even though only their url is entered in their records.

**Comment by droidmonkey** [comment, marked as answer]
Yes, go to the application settings, then browser tab, then enable "Only return best matching credentials"
